XamarinCommunityToolkit/Extensions/ImageResourceExtension.shared.cs

```diff
@@ -1,7 +1,7 @@
 ﻿using System;
 using Xamarin.Forms;
 using Xamarin.Forms.Xaml;
-#if NETSTANDARD1_0
+#if NETSTANDARD1_0 || UAP10_0
```

Comment:
is the `|| UAP10_0` necessary?

Reply:
Alas, not it isn't but we've merged it now. 

It was necessary in my initial pull request that targetted UAP10.0 because that only supports .NET Standard 1. Since I updated the target to UAP10.0.16299 we have support for all .NET Standard 2.0 features so I should have deleted this `|| UAP10_0 check`.

At time of commiting I did not know this but now I do. Relistcally it means almost nothing and will not stop anything from running correctly. I suppose I should open another PR as this code change is just unneeded complexity.